src/channel/ably/utils.ts

```diff
@@ -33,18 +33,37 @@ export const toTokenDetails = (jwtToken: string): TokenDetails | any => {
 
 const isBrowser = typeof window === 'object';
 
+/**
+ * Helper method to decode base64 url encoded string
+ * https://stackoverflow.com/a/78178053
+ * @param base64 base64 url encoded string
+ * @returns decoded text string
+ */
 export const toText = (base64: string) => {
```

Comment:
I would rename to `fromBase64UrlEncoded`. it should be clear that it's not regular base64 string expected